test/api/core/entrypoints/embedded/protocols/http.test.js

```diff
@@ -718,8 +744,28 @@ describe('/lib/api/core/entrypoints/embedded/protocols/http', () => {
         should(zlibstub.deflate).not.called();
         should(zlibstub.gzip).calledOnce();
       });
-    });
 
+      it('should overwrite the content-length even if one was set', () => {
+        protocol._sendRequest({id: 'connectionId'}, response, payload);
+        const
+          cb = kuzzle.router.http.route.firstCall.args[1],
+          request = new Request({}),
+          result = 'foobar';
+
+        request.setResult(result, {
+          raw : true
+        });
+
+        request.response.setHeader('content-length', 'one does not simply set a content length');
```

Comment:
You can wrap this line
```suggestion
        request.response.setHeader(
          'content-length',
          'one does not simply set a content length');
```